iceoryx_posh/test/moduletests/test_mepoo_memory_info.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 - 2022 by Apex.AI Inc. All rights reserved.
```

Comment:
Was this existing code?

Reply:
Nope, the header was just copied ... does this already count as copyright :smile: